test/robot_test.coffee

```diff
@@ -35,6 +35,7 @@ describe 'Robot', ->
 
     # Re-throw AssertionErrors for clearer test failures
     @robot.on 'error', (name, err, response) ->
+      return unless err?
```

Comment:
Hmm, the callback arity doesn't look is right. It should be `(err, response)`. `response` is optional, so maybe that's why it was failing? https://github.com/github/hubot/blob/receive-middleware/src/robot.coffee#L196-L202 for reference